node/src/effect/announcements.rs

```diff
@@ -224,16 +224,16 @@ pub(crate) enum ContractRuntimeAnnouncement {
     StepSuccess {
         /// The era id in which the step was committed to global state.
         era_id: EraId,
-        /// The operations and transforms committed to global state.
-        execution_effect: ExecutionEffect,
+        /// The JSON API format journal of transforms committed to global state.
+        json_execution_journal: JsonExecutionJournal,
```

Comment:
We shouldn't be using JSON for internal structures. It's only a representation type so internally we should be using domain types like `ExecutionJournal` and map it to a JSON only before presentation (here, even stream).